docs/testplan/token-payment-processor-testplan.md

```diff
@@ -0,0 +1,49 @@
+# Token Payment Processor Testplan
+
+&nbsp;
+
+## Background
+This testplan is going to cover the token payment processor, it will go over must haves and additional features that we can add on later. 
+
+UI relating to token payment processing can be seen on [Billing Page](https://www.figma.com/file/HlmasFJNHxs2lzGerq3WYH/Satellite-GUI-Public?node-id=11080%3A68109) figma design.
+
+&nbsp;
+
+&nbsp;
+
+<google-sheets-html-origin><!--td {border: 1px solid #ccc;}br {mso-data-placement:same-cell;}-->
+
+Test Scenario | Test Case | Description | Comments
+-- | -- | -- | --
+Transaction | Block Chain Reorg | If there is a naturally occurring chain reorg then past transactions in the old chain are deactivated due to a chain reorganization (this should be visible in UI, pending to cancelled) | Must have early on
+  | Transaction Override | If a user overrides the first transaction with a higher transaction fee, then the token payment processor has to replace the transaction hash it is watching (this should be visible in UI) | Must have early on
+  | Minimum Confirmation | A transaction should not be called paid unless it meets the minimum number of confirmations reached  (this should be visible in UI, pending until it meets minimum number of confirmations) | Must have early on
+  | Cancel Transaction- 0 Confirmations | If a user has a pending transaction and it has 0 confirmations, then user should be able to cancel transaction from their wallet (change should be visible on UI, Pending to cancelled) | Must have early on
+  | Cancel Transaction- Double Spending | If a user has a pending transaction and it has more than or equal to 1 confirmation, then user could override first transaction with a higher transaction fee and send to themself (this should be visible in UI, Pending to cancelled) | Must have early on
+  | Transaction Fee Warning | A general warning message should be shown in the UI regarding low transaction fees and explain to users that if the fee is too low then there is a possibility that the transaction will be reverted (their gas fee won't be given back) or delayed | Must have early on
+  | Multiple Transactions | If a user has a pending transaction and then performs another transaction using the same address, the new transaction should be placed on hold until the previous transaction is confirmed (this hold or pending status should show on the UI for any subsequent transactions using the same address) | Must have early on
```

Comment:
I don't think we have to worry about it. We can keep this test on the test plan just for documentation. At the end of the day, we just have 2 pending transactions. In order to block the second transaction, the first one needs to have a low fee. -> Duplicate to the low fee warning test case